package.json

```diff
@@ -15,8 +15,8 @@
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle static/js/build etc/",
     "serve": "./entrypoint $(hostname -I | awk '{print $1;}'):${PORT}",
     "start": "yarn run build && concurrently --raw 'yarn run watch-css' 'yarn run watch-js' 'yarn run serve'",
-    "cypress:run": "cypress run --config-file tests/cypress/cypress.json --config baseUrl=http://0.0.0.0:${PORT}",
-    "cypress:open": "cypress open --config-file tests/cypress/cypress.json --config baseUrl=http://0.0.0.0:${PORT}"
+    "cypress:run": "cypress run --config-file tests/cypress/cypress.json --config baseUrl=http://(hostname -I | awk '{print $1;}'):${PORT}",
```

Comment:
TOTO YOU FORGOT ABOUT THE `$` WHAT HAVE YOU DONE

Reply:
BLACE WHAT ARE YOU TALKING ABOUT?

Reply:
`baseUrl=http://(hostname -I | awk '{print $1;}'):${PORT}"` should be `baseUrl=http://$(hostname -I | awk '{print $1;}'):${PORT}"`